package.json

```diff
@@ -30,7 +30,7 @@
     "formik": "^2.2.9",
     "lodash": "^4.17.21",
     "moment": "^2.29.1",
-    "next": "^9.5.3",
+    "next": "^12.1.0",
```

Comment:
I forgot to remove `@babel/preset-env` and `@babel/preset-react`

Reply:
ok, i got it